.github/workflows/checks.yml

```diff
@@ -149,18 +152,24 @@ jobs:
           chmod u+x darwinia
           sudo mv darwinia /usr/bin
       - name: Launch darwinia
-        run: darwinia --chain pangolin-dev --tmp --alice &
+        run: darwinia --chain pangolin-dev --tmp --alice --tracing-api debug,trace &
```

Comment:
Need to update the node compile command with `evm-tracing` feature?

Reply:
Node is compiled with `--all-featuers`.